website/docs/library/restable.md

```diff
@@ -1,25 +1,239 @@
+|     8 |                                        0.737327598 |                       0.372048038 |        0.5046 |
+|     9 |                                        0.105332513 |                       0.053147195 |        0.5046 |
+|    10 |                                        0.015047502 |                       0.007592318 |        0.5046 |
+|    11 |                                        0.002149643 |                       0.001084609 |        0.5046 |
+|    12 |                                        0.000307092 |                       0.000154944 |        0.5046 |
+|    13 |                                        0.000043870 |                       0.000022135 |        0.5046 |
+|    14 |                                        0.000006267 |                       0.000003162 |        0.5046 |
+|    15 |                                        0.000000895 |                       0.000000452 |        0.5046 |
+
+*: Within a given resolution, all pentagons have the same area.
+
+
+### Average area in m<sup>2</sup>
+
+Here are the same areas, but in m<sup>2</sup>.
+
+|   Res |   Average <ins>Hexagon</ins> Area (m<sup>2</sup>) |   Pentagon Area* (m<sup>2</sup>) |
+|------:|--------------------------------------------------:|---------------------------------:|
+|     0 |                             4,357,449,416,078.392 |            2,562,182,162,955.496 |
+|     1 |                               609,788,441,794.134 |              328,434,586,246.469 |
+|     2 |                                86,801,780,398.997 |               44,930,898,497.879 |
+|     3 |                                12,393,434,655.088 |                6,315,472,267.516 |
+|     4 |                                 1,770,347,654.491 |                  896,582,383.141 |
+|     5 |                                   252,903,858.182 |                  127,785,583.023 |
+|     6 |                                    36,129,062.164 |                   18,238,749.548 |
+|     7 |                                     5,161,293.360 |                    2,604,669.397 |
+|     8 |                                       737,327.598 |                      372,048.038 |
+|     9 |                                       105,332.513 |                       53,147.195 |
+|    10 |                                        15,047.502 |                        7,592.318 |
+|    11 |                                         2,149.643 |                        1,084.609 |
+|    12 |                                           307.092 |                          154.944 |
+|    13 |                                            43.870 |                           22.135 |
+|    14 |                                             6.267 |                            3.162 |
+|    15 |                                             0.895 |                            0.452 |
+
+*: Within a given resolution, all pentagons have the same area.
```

Comment:
Ditto

Reply:
Did you really need to say that again? ;)